subsys/event_manager_profiler_tracer/event_manager_profiler_tracer.c

```diff
@@ -108,7 +143,16 @@ static void trace_register_events(void)
 	}
 }
 
-int event_manager_trace_event_init(void)
+/** @brief Initialize tracing in the Event Manager.
+ * The behavior of this function depends on the actual implementation.
+ * The default implementation of this function is no-operation.
```

Comment:
`not operational`?

Reply:
This is copy - paste leftover - to be fixed.

Reply:
Fixed. This is local documentation in C file, removed the whole detailed description.